tests/plugins/test_orchestrate_build.py

```diff
@@ -164,7 +164,24 @@ def mock_reactor_config(tmpdir, clusters=None, empty=False, add_config=None):
     if not clusters and not empty:
         clusters = deepcopy(DEFAULT_CLUSTERS)
 
-    conf_json = {'version': 1, 'clusters': clusters}
+    conf_json = {
+        'version': 1,
+        'clusters': clusters,
+        'odcs': {
+            'signing_intents': [
+                {
+                    'name': 'release',
+                    'keys': ['R123'],
+                    'deprecated_keys': ['R122']
+                }
+            ],
+            'default_signing_intent': 'release',
+            'api_url': 'http://odcs.example.com',
+            'auth': {
+                'ssl_certs_dir': '/var/run/secrets/atomic-reactor/odcssecret'
+            }
+        }
+    }
```

Comment:
Is the ODCS config required in all (or most) tests? Could it only be passed as the `add_config` param where necesary?